ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/validation/FieldValidations.scala

```diff
@@ -127,17 +127,17 @@ trait FieldValidations {
     */
   def validateDeduplicationPeriod(
       deduplicationPeriod: DeduplicationPeriodProto,
-      maxDeduplicationTimeO: Option[Duration],
+      optMaxDeduplicationTime: Option[Duration],
```

Comment:
We still have the "time" naming in the ledger configuration proto, being consistent with that but we may want rename it (ideally via deprecation).